doc/connecting-lab.md

```diff
@@ -12,44 +12,72 @@ document explains how to do this.
+
+Please add a new entry to the `runtimes` section of the configuration file
+as follows:
 
-The next step is to add the token to the pipeline services configuration files.
-Secrets are stored in the `kernelci.secrets` section of the `kernelci.toml` file,
-and added manually by the KernelCI system administrators.  For example, the token
-can be added to the `runtime.lava-labname` section of the `kernelci.toml` file:
+```yaml
+
+  lava-broonie:
+    lab_type: lava
+    url: 'https://lava.sirena.org.uk/'
+    priority_min: 10
+    priority_max: 40
+    notify:
+      callback:
+        token: kernelci-new-api-callback
+        url: https://staging.kernelci.org:9100
+
+```
+Where `lava-broonie` is the name of the lab, `lava` is the type of the lab, `url` is the URL of the lab, `priority_min` and `priority_max` are the priority range allowed to jobs, assigned by lab owner, and `notify` is the notification configuration for the lab.  The `callback` section contains the token description that you received from the above [step](#token-setup) and the URL of the pipeline instance LAVA callback endpoint.
+More details on how LAVA callback and token works can be found in the [LAVA documentation](https://docs.lavasoftware.org/lava/user-notifications.html#notification-callbacks).
+
+Please submit a pull request to [`kernelci-pipeline`](https://github.com/kernelci/kernelci-pipeline) repository to add the lab configurations. See the
+[pull request](https://github.com/kernelci/kernelci-pipeline/pull/426) for reference.
+
+### KernelCI configuration (TOML) file
+
+The next step is to add the token to the pipeline services configuration file i.e. [`config/kernelci.toml`](https://github.com/kernelci/kernelci-pipeline/blob/main/config/kernelci.toml) file. Every lab/runtime should have a section `runtime.<lab-name>` in the TOML file. The lab token should be stored in a key named `runtime_token` inside the section.
+For example,
 
 ```toml
-[runtime.<lava-labname>]
-runtime_token="lab-token-value"
+[runtime.<lab-name>]
+runtime_token="<lab-token-value>"
 ```
-Section name `lava-labname` should be replaced with the actual lab name, **matching the name of the lab in the pipeline configuration**.
-lab-token-value should be replaced with the actual token value. Usually it is a long string of random characters.
-For example in our documentation we use `lava-broonie` as the lab name, so the section will look like this
+
+Section name `lab-name` should be replaced with the actual lab name, **matching the name of the lab in the pipeline configuration i.e. `config/pipeline.yaml`**.
+`lab-token-value` should be replaced with the actual token value that you received in the [`Token setup`](#token-setup) step. Usually, it is a long string of random characters.
+For example, in our documentation we used `lava-broonie` as the lab name, so the section will look like this:
 ```toml
 [runtime.lava-broonie]
 runtime_token="M5d34U5pwCJnQLun"
```

Comment:
nit: maybe state explicitly that it's 100% **not** leaked token from c&p?

```suggestion
runtime_token="N0tAS3creTT0k3n"
```

Reply:
Thanks. I updated the token.